acceptance/openstack/autoscaling/v2/policies_test.go

```diff
@@ -23,10 +23,11 @@ func TestPolicyLifecycle(t *testing.T) {
 	if networkID == "" || vpcID == "" {
 		t.Skip("OS_NETWORK_ID or OS_VPC_ID env vars are missing but AS Policy test requires")
 	}
```

Comment:
Does it pass for you? Here's what I get:
```
 policies_test.go:80: Failure in policies_test.go, line 80: unexpected error "Bad request with: [PUT https://as.eu-de.otc.t-systems.com/autoscaling-api/v2/a720688fb87f4575a4c000d818061eae/scaling_policy/01a2d094-2786-4656-ba75-b6f0f7edc069], error message: {\"error\":{\"code\":\"AS.3041\",\"message\":\"The scaling resource ID in the AS policy is null.\"}}"
    policies_test.go:55: Attempting to delete AutoScaling Policy
    policies_test.go:57: Failure in policies_test.go, line 57: unexpected error "Resource not found: [DELETE https://as.eu-de.otc.t-systems.com/autoscaling-api/v1/a720688fb87f4575a4c000d818061eae/scaling_policy/], error message: {\"error_msg\":\"The API does not exist or has not been published in the environment\",\"error_code\":\"APIGW.0101\",\"request_id\":\"e65c50484b517302c484a64e26414a34\"}\n"
  ```

Reply:
uh... according to the doc, it should work and you can see eco/check-pos passed but...
https://docs.otc.t-systems.com/en-us/api/as/as_06_0105.html

maybe they changed something, anyway I will check it.

Reply:
now it should work
--- PASS: TestPolicyLifecycle (4.22s)